recipes/gmp/all/conanfile.py

```diff
@@ -53,7 +58,9 @@ def _configure_autotools(self):
                 configure_stats = os.stat(configure_file)
                 os.chmod(configure_file, configure_stats.st_mode | stat.S_IEXEC)
             configure_args = []
-            if self.options.disable_assembly:
+            if self.options.enable_fat:
+                configure_args.append("--enable-fat")
+            elif self.options.disable_assembly:
```

Comment:
Since these are deleted options you need to use `get_safe("name", False)` to ensure it works with different combinations